### PR TITLE
ES: EP 5

### PR DESCRIPTION
**Pull request by NozomiRem**

### CODE REVIEW

File: Story/Episode 5/st_052090.csv [hunk 1]
```diff
@@ -0,0 +1,26 @@
+name01#0,"""Hombre mayor"""
+name02#0,"""Mujer mayor"""
+name03#0,"""Mujer joven"""
+name04#0,"""???"""
```
Comment (alama) [inline Story/Episode 5/st_052090.csv]: keep the text `？？？`

File: Dialogue/Mid-Mission/co_040121.csv [hunk 2]
```diff
@@ -1,4 +1,4 @@
-name01#0,"""ラベルナ"""
+name01#0,"""Lavernat """
```
Comment (alama) [inline Dialogue/Mid-Mission/co_040121.csv]: Undo this

File: Dialogue/co_040127.csv [hunk 3]
```diff
@@ -1,7 +1,7 @@
 Title#0,"""『ランクアップとバトルコイン』について"""
 text0001#0,"""バトル終了時、勝敗に応じて<br>アリーナポイントが変動します。<br><br>アリーナポイントは勝ち続けるほど増えていき<br>自身の所属するランクが上がっていきます。<br><br><c ffff00>高いランクになるほど<br>貴重なアイテムと交換できるバトルコインが<br>多くもらえるようになります。<c>"""
 text0002#0,"""バトルアリーナで手に入れたバトルコインは<br><c ffff00>所属シップのショップエリア２Ｆの<br>バトルコイン交換ショップにて<br>様々なアイテムと交換することができます。<c><br><br>バトルアリーナで勝ち残り続けて<br>より高ランクを目指していきましょう。<br><c ffff00>\u3000※ランクの詳細はバトルカウンターの<br>\u3000\u3000\u3000「バトルアリーナの説明を聞く」から<br>\u3000\u3000\u3000確認することができます。<c>"""
-name01#0,"""ラベルナ"""
+name01#0,"""Lavernat """
```
Comment (alama) [inline Dialogue/co_040127.csv]: Undo this

File: Dialogue/co_050048.csv [hunk 4]
```diff
@@ -0,0 +1,6 @@
+Title#0,"""Acerca de 『Armas adicionales de Battle Arena』"""
+text0001#0,"""【Partizan】<br>Es un arma especializada en la lucha cuerpo a cuerpo<br>de alto HP y potencia de ataque.<br>Se puede utilizar las siguientes PAs.<br><c 00b0f0>\u3000・Assault buster<c><br>\u3000\u3000Ata la interrupción enemiga por un momento,<br>\u3000\u3000y realiza un ataque penetrante.<br>\u3000\u3000Aunque el rango de ataque en la dirección vertical<br>\u3000\u3000y la dirección horizontal es estrecho,<br>\u3000\u3000es excelente en el rango de ataque<br>en la dirección hacia delante."""
+text0002#0,"""【Twin machine gun】<br>Es un arma especializada en batalla de mediano alcance<br>combinando alta movilidad y ataque de disparos.<br><br>Mediante el uso de<c ffff00>Stylish roll<c><br>Mediante el uso, puede consumir PP <br>y tomar medidas rápidamente en la parte <br>delantera y trasera y la dirección izquierda y derecha.<br><br>Se puede utilizar la siguiente PA.<br><c 00b0f0>\u3000・Satellite Aim<c><br>\u3000\u3000Dispara 2 tiros consecutivos <br>\u3000\u3000Tiene menor daño pero tiene un<br>  rendimiento fácil de manejar."""
+name01#0,"""Lavernat """
```
Comment (alama) [inline Dialogue/co_050048.csv]: Undo this

File: Dialogue/pr_048010.csv [hunk 5]
```diff
@@ -1,5 +1,5 @@
 name01#0,"""？？？"""
-name02#0,"""ラベルナ"""
+name02#0,"""Lavernat """
```
Comment (alama) [inline Dialogue/pr_048010.csv]: Undo this

File: Misc/actor_name.csv [hunk 6]
```diff
@@ -631,7 +631,7 @@ Npc_390#0,"""イツキ"""
 Npc_266#0,"""アル"""
 Npc_PvCounterQuest1#1,"""グレトリ"""
 Npc_PvCounterQuest2#1,"""フリネ"""
-Npc_399#1,"""ラベルナ"""
+Npc_399#1,"""Lavernat """
```
Comment (alama) [inline Misc/actor_name.csv]: Undo this

**Review by alama — CHANGES_REQUESTED** [review]
let `ラベルナ` alone

File: Dialogue/Mid-Mission/co_040121.csv (hunk 2 above)
Comment (NozomiRem) [reply Dialogue/Mid-Mission/co_040121.csv]: the error is to the location

File: Dialogue/co_040127.csv (hunk 3 above)
Comment (NozomiRem) [reply Dialogue/co_040127.csv]: if I put """Lavernat""" in the future it gives me error when you rename

Comment (alama) [reply Dialogue/co_040127.csv]: Here how it goes, all patchs are made with EN files first, then we OVERLAY with the non-EN text

so just keep it as `ラベルナ`

**Review by alama — APPROVED** [review]

**Pull request merged**